conjure-java-core/src/integrationInput/java/com/palantir/product/EmptyUnionTypeExample.java

```diff
@@ -59,7 +59,7 @@ public String toString() {
     }
 
     public interface Visitor<T> {
-        T visitUnknown(String unknownType);
+        T visitUnknown(String unknownType, Map<String, Object> unknownValue);
```

Comment:
Same as the other PR:
```suggestion
        T visitUnknown(String unknownType, Object unknownValue);
```
unknownValue should be the result of `map.get(unknownType)`